File: 04) PHP Web Development Basics/10) OOP Basics - Exercises/9. Google/Car.php
<?php

namespace Van\ClassesA;

class Car
{
    /**
     * @var string
     */
    private $model;

    /**
     * @var int
     */
    private $speed;

    /**
     * Car constructor.
     *
     * @param string $model
     * @param int $speed
     */
    public function __construct(string $model, int $speed)
    {
        $this->model = $model;
        $this->speed = $speed;
    }

    /**
     * @return string
     */
    public function getModel(): string
    {
        return $this->model;
    }

    /**
     * @param string $model
     */
    public function setModel(string $model)
    {
        $this->model = $model;
    }

    /**
     * @return int
     */
    public function getSpeed(): int
    {
        return $this->speed;
    }

    /**
     * @param int $speed
     */
    public function setSpeed(int $speed)
    {
        $this->speed = $speed;
    }
}

File: 09) Java Advanced - January 2019/14. Defining Classes - Exercises/07. Google/Car/Car.java
package Car;

public class Car {

    private String model;

    private int speed;

    public Car(String model, int speed) {
        setModel(model);
        setSpeed(speed);
    }

    @Override
    public String toString() {
        return String.format("%s %d", model, speed);
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public int getSpeed() {
        return speed;
    }

    public void setSpeed(int speed) {
        this.speed = speed;
    }
}
